src/components/AddBookForm/AddBookForm.js

```diff
@@ -6,16 +6,19 @@ const AddBookForm = () => {
   const [title, setTitle] = useState();
   const [author, setAuthor] = useState();
   const dispatch = useDispatch();
+
   const addBookHandeler = (event) => {
     event.preventDefault();
     if (!title || !author) return;
     const book = {
       title,
       author,
-      id: Math.floor(Math.random() * 10000).toString(),
+      item_id: Math.floor(Math.random() * 10000).toString(),
+      category: 'Fiction',
```

Comment:
- [ ] (Optional) I would like to advise you to use `UUID` to create the id instead of using `Math.random()` because if you use `Math.random(),` there is a possibility that it will return the same id for the two different books. You can read this [article](https://www.npmjs.com/package/uuid) to learn how to use UUID. 👍